internal/install/recipe_installer.go

```diff
@@ -288,14 +290,45 @@ func (i *RecipeInstall) install(ctx context.Context) error {
 	return nil
 }
 
+func (i *RecipeInstall) printStartInstallingMessage(repo *recipes.RecipeRepository) {
+	message := "\n\nInstalling New Relic"
+	if i.RecipeNamesProvided() && len(i.RecipeNames) > 0 {
+		r := repo.FindRecipeByName(i.RecipeNames[0])
+		if r != nil {
+			message = fmt.Sprintf("%s %s", message, r.DisplayName)
+		}
+	}
+	fmt.Println(message)
+}
+
+func (i *RecipeInstall) reportRecipeStatuses(availableRecipes map[string]*recipes.RecipeDetectionResult,
+	unavailableRecipes map[string]*recipes.RecipeDetectionResult) {
+
+	for _, d := range unavailableRecipes {
+		e := execution.RecipeStatusEvent{Recipe: *d.Recipe, ValidationDurationMs: d.DurationMs}
+		i.status.ReportStatus(d.Status, e)
+	}
+
+	// Sort the key, so we can get consistent ordering
+	keys := make([]string, 0)
+	for k := range availableRecipes {
+		keys = append(keys, k)
+	}
+	sort.Strings(keys)
```

Comment:
Is this sort following the desired order? Infra first, then log...?

Reply:
We use the map here since we use available recipes for lookup in the bundler.  We can have the detector return an ordered slice, and I can update the bundler to have it convert to a map.  
Let me know if you think that's a better appraoch.

Reply:
Updated to have stable sort.